File: sidecars/synaplan-compute/tests/hostile/huge_stdout.py
# expected-result: succeeded
# reason:
# truncated-stdout: true
# note: Stream truncated server-side at the log cap; run finishes
"""huge_stdout.py — emit more than COMPUTE_LOG_CAP_BYTES of stdout."""
import sys

MARKER = "SYNAPLAN_HUGE_STDOUT_MARKER"
chunk = (MARKER + "\n") * 64
for _ in range(4096):
    sys.stdout.write(chunk)
sys.stdout.flush()

File: sidecars/synaplan-compute/tests/hostile/node/huge_stdout.js
# expected-result: succeeded
# reason:
# truncated-stdout: true
# note: Stream truncated server-side at the log cap; run finishes
"""huge_stdout.js — emit more than COMPUTE_LOG_CAP_BYTES of stdout."""
const MARKER = 'SYNAPLAN_HUGE_STDOUT_MARKER';
const chunk = (MARKER + '\n').repeat(64);
for (let i = 0; i < 4096; i++) {
  process.stdout.write(chunk);
}
